bower.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "bbc-vj-sharetools",
   "description": "BBC VJ Sharetools Module",
-  "main": "sharetools.js",
+  "main": "bin/",
```

Comment:
Main should be a list of paths not a folder. main is used by other tools like the bower-requirejs module to know which files to require. So a folder will break it.